package.json

```diff
@@ -1,32 +1,36 @@
 {
-  "name": "nanomsg",
-  "version": "0.4.2",
-  "description": "Node bindings for nanomsg",
-  "main": "lib/index.js",
+  "name":               "nanomsg",
+  "version":            "0.4.2",
+  "description":        "Node bindings for nanomsg",
+  "main":               "lib/index.js",
   "dependencies": {
-    "bindings": "1.1.1",
-    "nan": "~1.6.2"
+    "bindings":         "1.2.1",
+    "nan":              "~1.7.0"
```

Comment:
what?! way easier to read this way, check it out in your text editor, you don't agree?